plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1126,25 +1154,26 @@ void producer_plugin::plugin_initialize(const boost::program_options::variables_
          }
       }
 #endif
+   }
```

Comment:
this `}` should not have been moved up I think.

Reply:
That was on purpose. I needed the setting of _ro_* outside of `if ( my->_ro_thread_pool_size > 0 )` because they are used even when there is no active read-only thread pool.